http_client.rst

```diff
@@ -858,9 +861,10 @@ This component supports both the native PHP streams and cURL to make the HTTP
 requests. Although both are interchangeable and provide the same features,
 including concurrent requests, HTTP/2 is only supported when using cURL.
 
-``HttpClient::create()`` selects the cURL transport if the `cURL PHP extension`_
-is enabled and falls back to PHP streams otherwise. If you prefer to select
-the transport explicitly, use the following classes to create the client::
+The :method:`Symfony\\Component\\HttpClient\\HttpClient::create()` method
```

Comment:
Is this rendered like:

`create` or `HttpClient::create` like before?

Reply:
This is rendered as "create()"

Reply:
You're right! I updated accordingly :ok_hand: